src/main/java/view/HomeView.java

```diff
@@ -69,10 +71,17 @@ public HomeView(HomeViewModel homeViewModel) {
                 }
         );
 
-        myReviews = new JButton("My Reviews");
+        myReviewsButton = new JButton("My Reviews");
         bottomButtons.add(recommendations);
         bottomButtons.add(myWatchlists);
-        bottomButtons.add(myReviews);
+        bottomButtons.add(myReviewsButton);
+        // Add ActionListener to trigger the controller
+        // Add ActionListener to trigger the controller
+        myReviewsButton.addActionListener(evt -> {
+          final User user = homeController.getUser(homeViewModel.getState().getUsername());
```

Comment:
This creates a lot of unnecessary methods. You can just pass in the username to the myReviewsController and then in the My_ReviewsInteractor you can get the user from the database.